samples/GuestConfiguration/deployextension-installed-application-linux/azurepolicy.json

```diff
@@ -0,0 +1,287 @@
+{
+    "properties": {
+        "displayName": "Deploy VM extension to audit that an application is installed inside Linux VMs",
+        "mode": "Indexed",
+        "description": "Include this rule to deploy the VM extension for Microsoft Guest Configuration, the VM extension for Microsoft Azure Managed Service Identity, and the content required to audit that an application is installed inside Linux virtual machines. This policy should only be used along with its corresponding audit policy in an initiative/policy set. For more information on guest configuration policies, please visit http://aka.ms/gcpol",
+        "metadata": {
+            "category": "Guest Configuration",
+            "requiredProviders": [
+                "Microsoft.GuestConfiguration"
+            ]
+        },
+        "parameters": {
+            "ApplicationName": {
```

Comment:
"ApplicationNames" with a csv example if you are going to support multiple

Reply:
corrected